docs/topics/Gateway.md

```diff
@@ -373,13 +373,13 @@ Events are payloads sent over the socket to a client that correspond to events i
 | [Hello](#DOCS_TOPICS_GATEWAY/hello)                                                 | defines the heartbeat interval                                                                                                   |
 | [Ready](#DOCS_TOPICS_GATEWAY/ready)                                                 | contains the initial state information                                                                                           |
 | [Resumed](#DOCS_TOPICS_GATEWAY/resumed)                                             | response to [Resume](#DOCS_TOPICS_GATEWAY/resume)                                                                                |
-| [Reconnect](#DOCS_TOPICS_GATEWAY/reconnect)                                         | server is going away, client should reconnect to gateway and resume                                                              |
+| [Reconnect](#DOCS_TOPICS_GATEWAY/reconnect)                                         | server is going away, the client should reconnect to Gateway and resume                                                              |
```

Comment:
either add "the" everywhere or nowhere
also, since i'm making a comment here, gateway should not be capitalized, it is not a proper noun